src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContent.tsx

```diff
@@ -102,6 +124,10 @@ const ReleaseContent = () => {
     release.publication.releases.length +
     release.publication.legacyReleases.length;
 
+  const hasRelatedDashboardsSection =
+    release.relatedDashboardsSection &&
+    release.relatedDashboardsSection.content?.length > 0;
```

Comment:
The optional chain on `content` doesn't seem necessary as it shouldn't be nullable. This entire statement could actually be simplified to:
```ts
const hasRelatedDashboardsSection = !!release.relatedDashboardsSection?.content.length;
```

Reply:
Changed in both places.